webpack.config.js

```diff
@@ -125,7 +126,16 @@ module.exports = function(env = {}, options = {}) {
       new ExtractTextPlugin({
         filename: assetsSubDirectory + `css/${prefix}[name].css?[chunkhash]`, // NOTE: postcss-pipeline currently does not support query hash (https://github.com/mistakster/postcss-pipeline-webpack-plugin/issues/30)
         allChunks: true,
-      })
+      }),
+
+      // Cleans dist directory and removes specific unnecessary files
+      new WebpackCleanPlugin([
```

Comment:
- [ ] This is only needed in production. Please move inside `isProduction` condition (and `!hasWatcher` even better).